app/models/person.rb

```diff
@@ -108,6 +109,10 @@ class Person < ActiveRecord::Base # rubocop:disable Metrics/ClassLength
   i18n_setter :gender, (GENDERS + [nil])
   i18n_boolean_setter :company
 
+  enum second_factor_auth: [:no_second_factor, :totp]
```

Comment:
```suggestion
  enum second_factor_autentication: [:no_second_factor, :totp]
```
anstatt no_second_factor einfach nil verwenden falls nicht aktiv ?